doc/ref/configuration/master.rst

```diff
@@ -1520,6 +1520,74 @@ Jinja environment init variable ``lstrip_blocks``.
 
     jinja_lstrip_blocks: False
 
+.. conf_master:: jinja_comment_prefix
+
+``jinja_line_comment_prefix``
+-----------------------
+
+.. versionadded:: 2017.FIXME
```

Comment:
Yeah, definitely. When we decide on a release version, we search for occurrences of the codename and replace them, so leaving it as ``2017.FIXME`` will keep us from finding this when the time comes.